README.md

```diff
@@ -1,35 +1,37 @@
-# Membrane Template Plugin
+# Membrane ICE plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_ice_plugin.svg)](https://hex.pm/packages/membrane_ice_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_libnice_plugin/)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_ice_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_ice_plugin)
 
-This repository contains a template for new elements.
+Membrane plugin for ICE protocol.
 
-Check out different branches for other flavours of template.
+It enables establishing connection, sending and receiving messages using ICE protocol.
 
-It is part of [Membrane Multimedia Framework](https://membraneframework.org).
+This package uses [fake_turn] and is part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
-The package can be installed by adding `membrane_template_plugin` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `membrane_ice_plugin` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:membrane_template_plugin, "~> 0.1.0"}
+    {:membrane_ice_plugin, "~> 0.9.0"}
   ]
 end
 ```
 
 ## Usage
 
-TODO
+See [example_project] for example usage or refer to
+[hex.pm](https://hex.pm/packages/membrane_ice_plugin) for more details about how to interact with
+Sink and Source.
```

Comment:
This looks to be outdated. Let's refer to our EndpointBin where we are using this plugin